### PR TITLE
Shapely: update to 1.7.0, drop python2

### PR DESCRIPTION
**Pull request by karl-nilsson**
Testers welcome

### CODE REVIEW

**Comment by q66** [comment]
probably needs more reverts

**Comment by Chocimier** [comment]
Should depend on python, as it doesn't link to it anymore.

File: srcpkgs/python3-shapely/template [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+hostmakedepends="python3-setuptools python3-Cython"
+makedepends="python3-devel python3-numpy geos-devel"
+checkdepends="python3-pytest"
+short_desc="Library for manipulation and analysis of geometric objects for Python3"
+maintainer="Karl Nilsson <karl.robert.nilsson@gmail.com>"
+license="BSD-3-Clause"
+homepage="https://github.com/Toblerity/Shapely"
+distfiles="https://github.com/Toblerity/Shapely/archive/${version}.tar.gz"
+checksum=0f4283d5591578340e36f649736b496d25a793a45b9a2c93b128d2a856f2089b
+nocross="Build is crazy"
```
Comment (Piraty) [inline srcpkgs/python3-shapely/template:17]: lol

Comment (karl-nilsson) [reply srcpkgs/python3-shapely/template:17]: The cross-build fails to find the geos library.
I couldn't figure it out, but you're welcome to give it a shot.

Comment (Piraty) [reply srcpkgs/python3-shapely/template:17]: nah ;)

**Pull request merged**